### PR TITLE
Fix knex timeout errors

### LINKED ISSUE #1564: Fix knex timeout errors
### Actual behavior
Following timeout errors are logged while running job for updating/syncing database with `app-registry` repo.
```
43|lisk-se | 2023-04-03 07:10:00 002: 2023-04-03T07:10:00.001 INFO [metadata] Starting process to sync database with the repo.
43|lisk-se | 2023-04-03 07:11:00 009: 2023-04-03T07:11:00.009 ERROR [downloadRepository] Unable to sync changes due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?.
43|lisk-se | 2023-04-03 07:11:00 009: 2023-04-03T07:11:00.009 WARN [metadata] Refreshing blockchain application metadata failed due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
43|lisk-se | 2023-04-03 07:20:00 004: 2023-04-03T07:20:00.002 INFO [metadata] Starting process to sync database with the repo.
43|lisk-se | 2023-04-03 07:21:00 008: 2023-04-03T07:21:00.007 ERROR [downloadRepository] Unable to sync changes due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?.
43|lisk-se | 2023-04-03 07:21:00 008: 2023-04-03T07:21:00.008 WARN [metadata] Refreshing blockchain application metadata failed due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
43|lisk-se | 2023-04-03 07:30:00 002: 2023-04-03T07:30:00.001 INFO [metadata] Starting process to sync database with the repo.
43|lisk-se | 2023-04-03 07:31:00 009: 2023-04-03T07:31:00.008 ERROR [downloadRepository] Unable to sync changes due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?.
43|lisk-se | 2023-04-03 07:31:00 009: 2023-04-03T07:31:00.009 WARN [metadata] Refreshing blockchain application metadata failed due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
43|lisk-se | 2023-04-03 07:40:00 002: 2023-04-03T07:40:00.001 INFO [metadata] Starting process to sync database with the repo.
43|lisk-se | 2023-04-03 07:41:00 009: 2023-04-03T07:41:00.008 ERROR [downloadRepository] Unable to sync changes due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?.
43|lisk-se | 2023-04-03 07:41:00 009: 2023-04-03T07:41:00.008 WARN [metadata] Refreshing blockchain application metadata failed due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
43|lisk-se | 2023-04-03 07:50:00 003: 2023-04-03T07:50:00.002 INFO [metadata] Starting process to sync database with the repo.
43|lisk-se | 2023-04-03 07:51:00 008: 2023-04-03T07:51:00.007 ERROR [downloadRepository] Unable to sync changes due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?.
43|lisk-se | 2023-04-03 07:51:00 008: 2023-04-03T07:51:00.008 WARN [metadata] Refreshing blockchain application metadata failed due to: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) cal
```
### Expected behavior
Syncing process should complete without any errors

### Steps to reproduce

- Run `app-registry` service
- The above errors will be reported in the console when syncing database job executes

### Which version(s) does this affect? (Environment, OS, etc...)
v0.10.0-alpha-11

### PR DESCRIPTION
**Pull request by priojeetpriyom**
### What was the problem?
This PR resolves #1564

### How was it solved?
- [x] Use absolute ip (127.0.0.1) instead of `localhost` in mysql connection string
- [x]  Fix copied files are not owned by lisk user in docker
- [x] Update `docker-compose` version to 3.3
 
### How was it tested?
- [x] Local

### CODE REVIEW

File: services/blockchain-app-registry/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json ./
```
Comment (sameersubudhi) [inline services/blockchain-app-registry/Dockerfile:12]: This change seems unnecessary.

File: services/blockchain-coordinator/Dockerfile [hunk 2]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json ./.npmrc ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json ./.npmrc ./
```
Comment (sameersubudhi) [inline services/blockchain-coordinator/Dockerfile:12]: Same as above.

File: services/blockchain-indexer/Dockerfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json ./.npmrc ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json ./.npmrc ./
```
Comment (sameersubudhi) [inline services/blockchain-indexer/Dockerfile:12]: Same as above.

File: services/export/Dockerfile [hunk 4]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json ./
```
Comment (sameersubudhi) [inline services/export/Dockerfile:12]: Same as above.

File: services/fee-estimator/Dockerfile [hunk 5]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json ./.npmrc ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json ./.npmrc ./
```
Comment (sameersubudhi) [inline services/fee-estimator/Dockerfile:12]: Same as above.

File: services/gateway/Dockerfile [hunk 6]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json ./
```
Comment (sameersubudhi) [inline services/gateway/Dockerfile:12]: Same as above.

File: services/template/Dockerfile [hunk 7]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json ./
```
Comment (sameersubudhi) [inline services/template/Dockerfile:12]: Same as above.

File: tests/Dockerfile [hunk 8]
```diff
@@ -9,7 +9,7 @@ RUN adduser -D builder && \
 USER builder
 WORKDIR /home/builder/build
 
-COPY ./package-lock.json ./package.json .babelrc ./
+COPY --chown=lisk:lisk ./package-lock.json ./package.json .babelrc ./
```
Comment (sameersubudhi) [inline tests/Dockerfile:12]: Same as above.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Review by Nazgolze — APPROVED** [review]

**Review by nagdahimanshu — APPROVED** [review]

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**